description/launch/spawn.launch

```diff
@@ -28,6 +28,11 @@
         <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
         <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
         
+        <!-- Added rqt robot steering -->
+        <node name="rqt_robot_steering" pkg="rqt_robot_steering" type="rqt_robot_steering">
+        <param name="default_topic" value="/cmd_vel"/> 
```

Comment:
Change /cmd_vel, to /key_vel since key_vel is higher topic in rqt_graph

Reply:
we are not going to use `key_vel`. Rather, using `rqt_robot_steering` is basically manual control, so we want it to publish to `man_vel`.